Modules/Microsoft365DSC/DSCResources/MSFT_EXOAuthenticationPolicy/MSFT_EXOAuthenticationPolicy.schema.mof

```diff
@@ -0,0 +1,27 @@
+[ClassVersion("1.0.0.0"), FriendlyName("EXOAuthenticationPolicy")]
+class MSFT_EXOAuthenticationPolicy : OMI_BaseResource
+{
+    [Key, Description("The Identity parameter specifies the authentication policy you want to view or modify.")] String Identity;
+    [write, Description("The AllowBasicAuthActiveSync switch specifies whether to allow Basic authentication with Exchange Active Sync")] Switch AllowBasicAuthActiveSync;
+    [write, Description("The AllowBasicAuthAutodiscover switch specifies whether to allow Basic authentication with Autodiscover")] Switch AllowBasicAuthAutodiscover;
+    [write, Description("The AllowBasicAuthImap switch specifies whether to allow Basic authentication with IMAP.")] Switch AllowBasicAuthImap;
+    [write, Description("The AllowBasicAuthMapi switch specifies whether to allow Basic authentication with MAPI.")] Switch AllowBasicAuthMapi;
+    [write, Description("The AllowBasicAuthOfflineAddressBook switch specifies whether to allow Basic authentication with Offline Address Books.")] Switch AllowBasicAuthOfflineAdressBook;
```

Comment:
Change all Switch parameters to Boolean

Reply:
Changed switch parameters to system.boolean